lib/main.dart

```diff
@@ -128,13 +140,38 @@ class _MyHomePageState extends State<MyHomePage> {
                     emailField,
                     SizedBox(height: 25.0),
                     passwordField,
-                    SizedBox(
-                      height: 35.0,
-                    ),
+                    SizedBox(height: 25.0,),
                     loginButton,
-                    SizedBox(
-                      height: 15.0,
-                    ),
+                    SizedBox(height: 65.0,),
+                    registerButton,
+                   
+                   /*For demo purposes, these bypass the loginButton*/
+                  SizedBox(height: 30,),
+                  Row(
+                    mainAxisAlignment: MainAxisAlignment.end,
+                    children: <Widget>[
+                      RaisedButton(
```

Comment:
These two raised buttons are basically the same, can we pull this out into one widget definition that's called twice?